viewer/static/js/app/components/views/dataset-control.js

```diff
@@ -18,15 +33,7 @@ export default function(props) {
           })}
         </select>
       </div>
-      <div className="dataset-subgroups">
-        <select className="dataset-subgroup-selection" value={props.subgroup} onChange={props.handleSubgroupSelection}>
-          {sortedSubgroups.map(subgroup => {
-            return (
-              <option key={subgroup} value={subgroup}>{subgroup}</option>
-            );
-          })}
-        </select>
-      </div>
+      {maybeSubgroupSelector}
```

Comment:
Didn't know `null` is valid in JSX. Neat though.

Reply:
Yep. `{null}` will show nothing.